src/api/services/adminServiceClient.ts

```diff
@@ -20,6 +22,29 @@ const DEFAULT_SHARING_SETTINGS: Pick<SharingListResponse, 'allowed_sites' | 'all
   allowed_sites: [],
 };
 
+const mapClientTypesToAdminEnums = (
```

Comment:
nit: maybe this should live in a separate helper file, since it doesn't interact directly with admin?

Reply:
Good idea.  I moved it to adminServiceHelpers.ts